src/DynamoCoreWpf/Views/Search/LibraryContainerView.xaml

```diff
@@ -234,83 +281,140 @@
+               PlacementTarget="{Binding ElementName=ViewLayoutSelector}"
+               VerticalOffset="3"
+               HorizontalOffset="-10">
+            <ContentControl Style="{StaticResource LibraryPopupContentStyle}">
+                <WrapPanel>
+                    <!-- Compact layout button -->
+                    <Border BorderThickness="0,0,1,0"
+                            BorderBrush="{StaticResource SearchDarkGreyTextColor}">
+                        <Button Name="CompactButton"
+                                Command="{Binding ToggleLayoutCommand}">
+                            <Button.CommandParameter>
+                                <system:Boolean>False</system:Boolean>
+                            </Button.CommandParameter>
+                            <Button.Template>
+                                <ControlTemplate>
+                                    <StackPanel Orientation="Horizontal"
+                                                Margin="5,5,5,5">
+                                        <fa:ImageAwesome Name="CompactLayoutIcon"
+                                                         Icon="list"
+                                                         Foreground="{Binding IsDetailedMode, 
+                                                                              Converter={StaticResource CompactLayoutForegroundConverter}}"
+                                                         Margin="5,0,5,0"
+                                                         Height="13"
+                                                         Width="13" />
+                                        <TextBlock Foreground="{Binding IsDetailedMode, 
+                                                                              Converter={StaticResource CompactLayoutForegroundConverter}}"
+                                                   Text="{x:Static p:Resources.CompactLayoutTitle}"
+                                                   FontSize="13" />
+                                    </StackPanel>
+                                </ControlTemplate>
+                            </Button.Template>
+                        </Button>
+                    </Border>
+
+                    <!-- Detailed layout button -->
+                    <Button Name="DetailedButton"
+                            Command="{Binding ToggleLayoutCommand}">
+                        <Button.CommandParameter>
+                            <system:Boolean>True</system:Boolean>
+                        </Button.CommandParameter>
+                        <Button.Template>
+                            <ControlTemplate>
+                                <StackPanel Orientation="Horizontal"
+                                            Margin="5,5,5,5">
+                                    <fa:ImageAwesome  Name="DetailedLayoutIcon"
+                                                      Foreground="{Binding IsDetailedMode, 
```

Comment:
We need 2 opposite converters. Since we are using 1 property for 2 buttons.
I.e. if `IsDetailedMode` is true, then icon and "Detailed" textblock should be light grey. At the same time, when `IsDetailedMode` is true, icon and "Compact" textblock should be dark grey.
And vice versa.
![image](https://cloud.githubusercontent.com/assets/8158404/10759916/c40368f4-7cc2-11e5-92e5-63307371303a.png)